DataFormats/CaloRecHit/interface/CaloRecHit.h

```diff
@@ -4,27 +4,65 @@
 #include "DataFormats/DetId/interface/DetId.h"
 #include <ostream>
 
+namespace {
```

Comment:
this should not be used in header files

Reply:
ok, did not know... thx

i renamed, as __constant__ (memory specifier) can not be marked on a class member